lib/aux_js.cpp

```diff
@@ -208,9 +226,22 @@ void enableKeyHandling()
    SDL_EventState(SDL_KEYUP, SDL_ENABLE);
 }
 
-void setKeyboardListeningElementId(const std::string & elem_id)
+void setKeyboardListeningElementId(const std::string & id)
 {
-   SDL_SetHint(SDL_HINT_EMSCRIPTEN_KEYBOARD_ELEMENT, elem_id.c_str());
+   SDL_SetHint(SDL_HINT_EMSCRIPTEN_KEYBOARD_ELEMENT, id.c_str());
+}
+
+void setupResizeEventCallback(const std::string & id) {
+  // typedef EM_BOOL (*em_ui_callback_func)(int eventType, const EmscriptenUiEvent *uiEvent, void *userData);
+  std::cout << "glvis: adding resize callback for " << id << std::endl;
+  auto err = emscripten_set_resize_callback(id.c_str(), nullptr, true, [](int eventType, const EmscriptenUiEvent *uiEvent, void *userData) -> EM_BOOL {
```

Comment:
wrap at 80 lines plz :)

Reply:
Thanks for the `make style`! I can't get a copy of Astyle 2.05.1 for mac, the link on [this page](https://sourceforge.net/projects/astyle/files/astyle/astyle%202.05.1/) just ends up redirecting to the "Files" page. Any other places you know of to get a copy?

Reply:
That link actually downloads for me but you can also get it (and other TPLs) from here: https://mybitbucket.llnl.gov/projects/MFEM/repos/tpls/browse